Doc/library/functions.rst

```diff
@@ -96,7 +96,7 @@ are always available.  They are listed here in alphabetical order.
    This calls the :meth:`~object.__anext__` method of *async_iterator*,
    returning an :term:`awaitable`. Awaiting this returns the next value of the
    iterator. If *default* is given, it is returned if the iterator is exhausted,
-   otherwise :exc:`StopAsyncIteration` is raised.
+   otherwise, :exc:`StopAsyncIteration` is raised.
```

Comment:
The usage before was correct. If you insist on using the comma when "otherwise" begins a sentence fragment, you need to change the preceding comma into a semicolon to emphasize the composition of the sentence (it looks too similar to enumeration otherwise). I would just leave those cases alone.